qtpy/__init__.py

```diff
@@ -211,6 +211,13 @@ class PythonQtWarning(Warning):
 except (ImportError, PythonQtError):
     pass
 
+try:
+    # QtCharts backward compatibility with QtPy 1.x
+    from . import QtCharts
+    QtCharts.QtCharts = QtCharts
+except (ImportError, PythonQtError):
+    pass
```

Comment:
I agree with @CAM-Gerlach: can this manipulation be done in the QtCharts module instead of here?

Reply:
:point_up_2: ?

Reply:
Not sure, I will need to check

Reply:
Looks good!